channel.go

```diff
@@ -213,6 +217,23 @@ func (c *ChannelService) UpdateChannelMemberNotifications(channelID, userID stri
 	return channelMember, normalizeAppErr(appErr)
 }
 
+func (c *ChannelService) waitForChannelCreation(channelID string) error {
+	now := time.Now()
```

Comment:
Can we additionally gate this on a check for a replica, like the server code does?

Reply:
Oh whoops forgot we had access to the config. Check added.